addon/components/license-picker/component.js

```diff
@@ -0,0 +1,22 @@
+import Ember from 'ember';
+import layout from './template';
+
+export default Ember.Component.extend({
```

Comment:
Please make sure to see a YUIDoc docstring, including sample usage, for all ember-osf components. Otherwise this component will be invisible to the [documentation](http://centerforopenscience.github.io/ember-osf/).

See other components in this repo for an example and let me know if I can help. To verify this works correctly, you can run `npm run docs` with YUIDoc installed.